types/lib/associations/base.d.ts

```diff
@@ -1,9 +1,9 @@
 import { ColumnOptions, Model } from '../model';
 
-export abstract class Association {
+export abstract class Association<S = typeof Model, T = typeof Model> {
```

Comment:
Can you please add an export to `model.d.ts` that's called `ModelType = typeof Model` and use that instead?

Reply:
done